index.js

```diff
@@ -3,16 +3,24 @@ import { View, Platform } from 'react-native';
 
 export default class ElevatedView extends React.Component {
   static propTypes = {
-    //elevation of 0 results in a default View.
-    elevation: React.PropTypes.oneOf([0, 1, 2, 3, 4, 5])
-  }
+    elevation: React.PropTypes.number,
+  };
   static defaultProps = {
     elevation: 0
-  }
+  };
 
   render() {
     const { elevation, style, ...otherProps } = this.props;
 
+    //Return default View if elevation is 0
```

Comment:
Let's put this one below the `(Platform.OS === 'android')`, so it only effects iOS.

Also one nit: don't think we need the comment